source/includes/wp-api-v3/_orders.md

```diff
@@ -10,7 +10,7 @@ The orders API allows you to create, view, update, and delete individual, or a b
-| `created_via`          | string    | Shows where the order was created. <i class="label label-info">read-only</i>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| `created_via`          | string    | Shows where the order was created.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
```

Comment:
We should mention that the property population has its caveats: `Created via is only writable on order creation.` as [er https://github.com/woocommerce/woocommerce/pull/57225

Reply:
Good call, done!

Reply:
Nice, thanks!